src/App.tsx

```diff
@@ -1,26 +1,209 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoFilter } from './types/TodoFilter';
+import classNames from 'classnames';
+import { TodoList } from './TodoList';
+import { TodoFooter } from './TodoFooter';
+import { TodoForm } from './TodoForm';
+import * as todoServices from './api/todos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [filter, setFilter] = useState<TodoFilter>(TodoFilter.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [loadingId, setLoadingId] = useState<number | number[]>(0);
+
+  // #region loadTodos
+  async function loadTodos() {
```

Comment:
It is important to follow the same way of creating functions everywhere, fix it everywhere
```suggestion
  const loadTodos = async () => {
```